.dockerignore

```
@@ -1,3 +1,8 @@
-nss
-nspr
-target
+# Ignore everything:
+*
+# Except for the following:
+!**/*.toml
```

```suggestion
!**/*.toml
!Cargo.lock
```

Easier for debugging, as both a local `cargo xxx` as well as the `cargo build`  in the `Dockerfile` will use the same dependencies.